data_validation/clients.py

```diff
@@ -143,15 +124,15 @@ def get_ibis_table(client, schema_name, table_name, database_name=None):
     table_name (str): Table name of table object
     database_name (str): Database name (generally default is used)
     """
-    if type(client) in [
-        OracleClient,
-        PostgreSQLClient,
-        DB2Client,
-        MSSQLClient,
-        RedShiftClient,
+    if client.name in [
+        "oracle",
```

Comment:
Yeah their defined in Ibis itself - the new upgrade uses a Backend class instead of a Client class, and each Backend has a name property indicating the DB type.